src/main/java/org/mockito/internal/creation/MockSettingsImpl.java

```diff
@@ -239,7 +241,16 @@ public <T2> MockCreationSettings<T2> buildStatic(Class<T2> classToMock) {
 
     @Override
     public MockSettings lenient() {
-        this.lenient = true;
+        this.strictness = Strictness.LENIENT;
+        return this;
+    }
+
+    @Override
+    public MockSettings strictness(Strictness strictness) {
+        this.strictness = strictness;
+        if (strictness == null) {
+            throw strictnessDoesNotAcceptNullParameter();
```

Comment:
This line isn't covered yet by the tests. Therefore, let's add a test that attempts to set the `strictness` to `null` and ensures that an Exception is thrown.

Reply:
Added test to verify this.